release_notes/unreleased.md

```diff
@@ -1 +1,2 @@
 **Unreleased**
+* De-certifying and releasing as a community app
```

Comment:
do we also need to add a bullet point mentioning it was converted to python 3?

Reply:
Agree, done 
Thanks!